articles/node-vs-nuxt/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/node-vs-nuxt/
+title: Node vs Nuxt - The Key Differences
+description: .
+author: saiharsha-balasubramaniam
+date: 2020-09-06T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/node-eslint/hero.jpg
+    alt: linters example image ESLint
+---
+
+A while ago, traditional server side rendering used to be the norm. All of the HTML is dynamically rendered on the server and sent to the client. Back in those days, websites weren't user-friendly and native.
+
+Today's web consists of single-page applications and universal web-apps. Let us look at the differences between Node.js and Nuxt which are two important technologies used for the web.
+
+<!--more-->
+
+## Table of Contents
+
+- [Why is Node.js Needed](#why-is-node.js-needed)
+- [Why is Nuxt Needed](#why-is-nuxt-needed)
+- [Node.js-vs-Nuxt](#node.js-vs-nuxt)
+- [Create a Node.js App](#create-a-node.js-app)
+- [Create a Nuxt App](#create-a-nuxt-app)
+- [Further Reading](#further-reading)
```

Comment:
Consider adding a section about When to Use Node.js vs Nuxt which talks about ideal scenarios of using each which would guide readers more on which they should choose.